src/library/scala/collection/mutable/ListBuffer.scala

```diff
@@ -105,7 +105,7 @@ class ListBuffer[A]
     aliased = false
   }
 
-  def addOne(elem: A): this.type = {
+  final def addOne(elem: A): this.type = {
```

Comment:
@szeiger Did we consider making `ListBuffer` final in the 2.13.x collections rework?

Reply:
Not directly but it makes sense. We should mark it `@deprecatedInheritance` like we did for some other collection types. Most concrete collection implementations are not designed for extensibility.